tests/test_arithmetics.rs

```diff
@@ -423,3 +437,57 @@ fn rich_comparisons_python_3_type_error() {
     py_expect_exception!(py, c2, "c2 >= 1", PyTypeError);
     py_expect_exception!(py, c2, "1 >= c2", PyTypeError);
 }
+
+#[test]
+fn rich_comparisons_python3_not_implemented() {
+    let gil = Python::acquire_gil();
+    let py = gil.python();
+
+    let c2 = PyCell::new(py, RichComparisonToSelf {}).unwrap();
+    for operation in vec!["==", "<", ">", "<=", ">="] {
```

Comment:
We don't need vec here:
```suggestion
    for operation in ["==", "<", ">", "<=", ">="].iter() {
```

Reply:
Thanks @kngwyu 

I'm still a newly born rustacean ;)